src/PrestaShopBundle/Service/DataProvider/Marketplace/ApiClient.php

```diff
@@ -50,6 +51,7 @@ public function __construct(
 
     public function setSslVerification($verifySsl)
     {
+        Tools::refreshCACertFile();
```

Comment:
I'm not easy to rely on ``Tools`` class in a new class, do we have an adapter that you can inject instead ?